src/main/java/com/endava/internship/collections/Student.java

```diff
@@ -1,11 +1,16 @@
 package com.endava.internship.collections;
 
+import lombok.Getter;
+import lombok.Setter;
+
 import java.time.LocalDate;
+import java.util.Objects;
 
 /**
  * The class that defines the element that will be contained by your collection
  */
-public class Student //TODO consider implementing any interfaces necessary for your collection
+@Getter @Setter
+public class Student implements Comparable<Student> //TODO consider implementing any interfaces necessary for your collection
 {
     private String name;
     private LocalDate dateOfBirth;
```

Comment:
Before pushing your changes go to every file and press Ctrl + Alt + L to format the code